<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModeSelectionView</class>
 <widget class="QWidget" name="ModeSelectionView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Selección de Quiz</string>
  </property>
  <property name="styleSheet">
   <string>QWidget {
	background-color: #2e2e2e;
	color: white;
}
/* Estilo para los GroupBox (contenedores de modo y categoría) */
QGroupBox {
    border: 1px solid #555555;
    border-radius: 5px;
    margin-top: 20px;
    font-size: 16px;
    font-weight: bold;
    color: white;
}
QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 10px;
    color: #007bff;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <item>
    <widget class="QLabel" name="lbl_main_title">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string>color: #007bff; margin-top: 10px;</string>
     </property>
     <property name="text">
      <string> SELECCIÓN DE TRIVIA </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="selection_layout">
     <property name="spacing">
      <number>30</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_left">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="mode_group">
       <property name="title">
        <string>1. Modo de Juego</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="rb_clasica">
          <property name="objectName">
           <string>rb_clasica</string>
          </property>
          <property name="text">
           <string>Trivia Clásica (General)</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_tematica">
          <property name="objectName">
           <string>rb_tematica</string>
          </property>
          <property name="text">
           <string>Modo Temático (Por Categoría)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="category_group">
       <property name="title">
        <string>2. Elegir Categoría</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QComboBox" name="cmb_category">
          <property name="objectName">
           <string>cmb_category</string>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string>QComboBox { 
    background-color: #555555; 
    color: white; 
    padding: 5px; 
    border-radius: 3px; 
} 
QComboBox QAbstractItemView { 
    background-color: #555555; 
    color: white; 
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_right">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_bottom_fill">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>190</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="controls_layout">
     <property name="spacing">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_back">
       <property name="objectName">
        <string>btn_back</string>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string>background-color: #555555; color: white; border-radius: 8px;</string>
       </property>
       <property name="text">
        <string>← Volver al Menú</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_start_quiz">
       <property name="objectName">
        <string>btn_start_quiz</string>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string>background-color: #28a745; color: white; border-radius: 8px;</string>
       </property>
       <property name="text">
        <string>INICIAR QUIZ</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>